benchmark/src/test/java/com/datatorrent/benchmark/util/serde/GenericSerdePerformanceTest.java

```diff
@@ -7,7 +7,7 @@
  * "License"); you may not use this file except in compliance
  * with the License.  You may obtain a copy of the License at
  *
- *   http://www.apache.org/licenses/LICENSE-2.0
```

Comment:
This change does not look right, see license header in other files (and header.txt).